data/data-pipeline/data_pipeline/etl/sources/ncld_nature_deprived/etl.py

```diff
@@ -1,77 +0,0 @@
-# pylint: disable=unsubscriptable-object
```

Comment:
I'm not sure how we got to this state but we had one folder called `ncld_*` and one called `nlcd_*`. I deleted the one we're not using that has the typo.